include/TCL_obj_templates.h

```diff
@@ -361,6 +350,16 @@ namespace ecolab
     *memberptr=s;
     tclreturn() << s;
   }
+  
+  // deal with special float values. for ticket 1253
+  template <> inline void member_entry<double>::put(const char *s)
+  {
+    assert(memberptr);
+    try {
+		*memberptr=std::stod(s);
+		tclreturn() << std::stod(s);
+	} catch(...) {}
```

Comment:
Probably should tclreturn << 0.0 here.

Reply:
Or move the tclreturn outside the try/catch. It probably makes more sense for
tclreturn() << *memberptr

as the final statement of the function.

Reply:
Oops, I didn't test this one locally, because I made these ecolab commits online, not from the command line of my laptop. My latest commit throws a floating point exception and crashes Minsky when I try to run Steve's model... So, I'm going to have to try the first suggestion you made (putting tclreturn() << "0" in the catch statement).

Reply:
Sorry, there's no problem. The floating point exception was due to changes I started making towards feature 147 in minsky.cc. I was testing this on my develop branch, not a branch specific to bug 1253. I guess that all is well that ends well...